common/djangoapps/oauth_exchange/views.py

```diff
@@ -35,3 +43,26 @@ def post(self, request, _backend):
             edx_access_token = self.create_access_token(request, user, scope, client)
 
         return self.access_token_response(edx_access_token)
+
+
+class SessionCookieExchangeView(APIView):
+    """
+    View for exchanging an access token for session cookies
+    """
+    authentication_classes = (OAuth2AuthenticationAllowInactiveUser,)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    @method_decorator(csrf_exempt)
+    def get(self, request):
```

Comment:
The client isn't actually retrieving a resource but rather is making the request for the purpose of a side effect (namely the association of a user with a session cookie).